esm/shared/dictionary.js

```diff
@@ -0,0 +1,7 @@
+const {create} = Object;
+
+export class Dictionary {
+  constructor() {
+    this.prototype = create(null);
```

Comment:
nope ... this is not what I meant ... `Dictionary.prototype = create(null);` is what I was after, but on a closer look, we might want to get rid of this class completely and use a `Map` instead for listeners, so we don't clash, ever, and we have Map methods to deal with listeners in a natural way.

Reply:
I tried your initial suggestion first but that wasn't working and that version was the only way I could find. 
Will update with Map!